src/portable/nordic/nrf5x/dcd_nrf5x.c

```diff
@@ -118,7 +119,7 @@ static struct {
   xfer_td_t xfer[EP_CBI_COUNT + 1][2];
 
   // nRF can only carry one DMA at a time, this is used to guard the access to EasyDMA
-  atomic_bool dma_running;
+  atomic_flag dma_running;
```

Comment:
clang is picky, only alow using atomic_flag_ function with atomic_flag and not atomic_bool